### DISCUSSION TITLE
CVE-2023 4863

### DISCUSSION THREAD

**Discussion by holzi-x** [standardnotes/forum#3297] [General] [answered]
I SN affected to this vulnerability as it is stated [here](https://gist.github.com/mttaggart/02ed50c03c8283f4c343c3032dd2e7ec)
If so - when will a patch be available?

**Comment by effieeee** [comment]
Hi, it looks like the list in your link is an outdated copy, and the repository indicated on their [updated](https://github.com/mttaggart/electron-app-tracker/blob/main/electron_apps.csv) list is an older, archived repository of ours 🙏

**Reply by holzi-x** [reply]
Thanks for the update - my question is however still not answered. 
Is sn vulnerable to this threat?

**Comment by holzi-x** [comment, marked as answer]
As the SN Team seems to be overloaded I looked for the answer myself.
My current SN installation is latest and greatest V3.175.2 and is based on Electron V22.3.24
According to https://www.electronjs.org/blog/breach-to-barrier Electron versions <22.3.24 are affected by the bug. So all good here.
HOWEVER: At the time I posted my question, the Electron release was 22.1.0 (SN version 3.173.0 I think) and this was vulnerable. My feeling is, that the SN team has not been very transparent about this, which is a shame. It definitely does not help you to build trust. 
Remember: trust is a thing that builds up slowly and can be destroyed in a second, so be aware...

Anyway thanks for the electron update!
